src/components/styles/fonts.ts

```diff
@@ -0,0 +1,112 @@
+import { css } from "@emotion/react";
```

Comment:
Perhaps we should omit the Euclid Light and Bold font files since they're not registered via `@font-face` (which is fine — LeafyGreen [doesn't include them](https://github.com/mongodb/leafygreen-ui/blob/c26d1bec13a4fe18612c4a0e36c835cd83e454ef/website/styles/globals.tsx) either)

Reply:
Thanks just did some copypasta from the mms repo :) I should have checked